src/sync/installation.ts

```diff
@@ -245,7 +250,14 @@ async function doProcessInstallation(app, queues, job, installationId: number, j
 		installationId,
 		logger
 	);
+	const useNewGHClient = await booleanFlag(BooleanFlags.USE_NEW_GITHUB_CLIENT__FOR_PR, false, jiraHost);
```

Comment:
So would we have one feature flag for all? 

If yes then it will be better to call it differently USE_NEW_GITHUB_CLIENT

Otherwise we should evaluate within each processor